doc/source/getting_started/intro_tutorials/09_timeseries.rst

```diff
@@ -386,4 +386,4 @@ A full overview on time series is given in the pages on :ref:`time series and da
 
 .. raw:: html
 
-   </div>
```

Comment:
Hmm what changed here? Could you revert this diff?

Reply:
The editor (I used the GitHub website) appended a newline here, so that the file ends with a newline. I'd suggest to leave it in, otherwise the next person trying to edit this file with GitHub will run into the same issue.

Or would you prefer a separate PR to fix the file endings? It looks like most, but not all, other files in the Pandas docs already end with a newline. (Some also with multiple newlines...)

Reply:
this is ok